contracts/DebtLocker.sol

```diff
@@ -48,46 +48,46 @@ contract DebtLocker is IDebtLocker {
     function claim() external override isPool returns (uint256[7] memory) {
 
         uint256 newDefaultSuffered   = uint256(0);
-        uint256 loan_defaultSuffered = loan.defaultSuffered();
+        uint256 loan_defaultSuffered = ILoanLike(loan).defaultSuffered();
```

Comment:
I think we determined before that it was less gas efficient to cast immutable addresses to interfaces inline, maybe we should compare this to caching to a local interface variable. I also think this is pretty out of scope for this PR.

Reply:
Casting to an interface is free. Its actually wasteful to declare a variable that is a interface-casted version of an address as a separate stack variable. Also, part of a complete test accounts is such that no function requires or returns an interface, or else we are going to have problems in integration testing.

Reply:
Bytecode is going to be the same. But I can test to make sure.